src/components/ExploreCarousel/index.js

```diff
@@ -0,0 +1,159 @@
+import React from "react";
+import { H3, H2, StyledLink } from "../../styles/styles";
+import {
+  FlexContainer,
+  responsiveWatch,
+  responsiveRead,
+  FlexRow,
+  Work,
+  LeftIcon,
+  RightIcon,
+  StyledP,
+  StyledImage,
+  StyledArticleImage,
+  ItemContainer,
+} from "./styled";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+import { PrismicRichText } from "@prismicio/react";
+
+const ExploreCarousel = ({ section }) => {
+  const {
+    data: { title, works },
+    uid: sectionuid,
+  } = section;
+
+  const CustomLeftArrow = ({ onClick }) => (
+    <LeftIcon onClick={() => onClick()} />
+  );
+  const CustomRightArrow = ({ onClick }) => {
+    return <RightIcon onClick={() => onClick()} />;
+  };
+
+  const SingleCarousel = () => (
+    <Carousel
+      responsive={responsiveWatch}
+      customRightArrow={<CustomRightArrow />}
+      customLeftArrow={<CustomLeftArrow />}
+      infinite={true}
+    >
+      {works.map((work) => {
+        const {
+          work: {
+            document: {
+              uid,
+              data: { title, thumbnail, preview_text: previewText },
+            },
+          },
+        } = work;
+        return (
+          <ItemContainer>
+            <StyledLink to={`../works/${uid}`}>
+              <Work key={uid}>
+                <StyledImage alt={title.text} src={thumbnail.url} />
+                <H3>{title.text}</H3>
+                <PrismicRichText
```

Comment:
here, and anywhere else where there is `<PrismicRichText>` with just a `<StyledP>`:
in conjunction with my previous comment, once you change `richText` to `text` you can convert all these rich text components to simply a `<StyledP>`